packages/shared/components/FormLogin/FormLogin.tsx

```diff
@@ -300,7 +288,7 @@ type Props = {
   auth2faType?: Auth2faType;
   attempt: ReturnType<typeof useAttempt>[0];
   isRecoveryEnabled?: boolean;
-  onRecover?: (type: 'totp' | 'u2f' | 'password') => void;
+  onRecover?: (isRecoverPassword: boolean) => void;
```

Comment:
keep the strings as it's way more descriptive than a boolean flag. -> `password | 2fa`